templates/zerver/help/search-for-messages.md

```diff
@@ -3,69 +3,125 @@
 Zulip has a powerful search engine under its hood. Search for messages using
 the search bar at the top of the app.
```

Comment:
Maybe we should tweak the first couple of sentences and say something like:

```suggestion
Zulip has a powerful search engine under its hood. You can search for messages using
keywords and filters via the search bar at the top of the app.
```

Reply:
I was also wondering if adding a keyboard tip for the search shortcuts would make sense here (or elsewhere on the page, maybe at the end of the keyword search section?).

```
!!! keyboard_tip ""

    You can activate the search bar by typing <kbd>/</kbd> or
    <kbd>Ctrl</kbd> + <kbd>K</kbd>.
```